### PR TITLE
ELEMENTS-220: introduce nuxeo-layout

### PR DESCRIPTION
**Pull request by nelsonsilva**
Introduces `nuxeo-layout` and brings in `nuxeo-error` for convenience.

### CODE REVIEW

File: nuxeo-error.html [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+@group Nuxeo UI
+@element nuxeo-error
+-->
+<dom-module id="nuxeo-error">
+  <template>
+    <style>
+       :host {
+        display: block;
+        margin: 16px;
```
Comment (adilioaraujo) [inline nuxeo-error.html]: Not a big fan of having margins on UI elements (I know it was there in the context of the Web UI)

File: nuxeo-error.html [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+        font-weight: 500;
+      }
+    </style>
+    <div class="code" hidden$="[[!code]]">[[code]]</div>
+    <div class="description" hidden$="[[!code]]">[[_label(code)]]</div>
+    <div class="url" hidden$="[[!url]]">[[url]]</div>
+    <div class="message" hidden$="[[!message]]">[[message]]</div>
+  </template>
+  <script>
+    (function() {
```
Comment (adilioaraujo) [inline nuxeo-error.html]: Function wrapper is really needed?

**Comment by guirenard** [comment]
:+1:

File: nuxeo-error.html (hunk 1 above)
Comment (nelsonsilva) [inline nuxeo-error.html]: margin was in a `main` container which I have since removed, should I just remove the margin ?

**Pull request merged**